scripts/test-runner.js

```diff
@@ -175,13 +176,7 @@ if (process.argv.length === 3 || process.argv.length === 4) {
         }
         testType = 'all';
     } else if (process.argv[2] === 'startrc') {
-        startRC(true).then(() => {
```

Comment:
This was buggy. On windows I realized that startrc() was called twice in `startrc` operation. Fixed it